pytripgui/plan_executor/simulation_results.py

```diff
@@ -10,52 +10,50 @@
 class SimulationResults:
     def __init__(self, patient, plan):
         self.patient = patient
-        self.name = ""
-        self.plan = None
-        self.dose = None
-        self.let = None
+        self.name = plan.basename
+        self.plan = plan
         self.volume_histograms = {}
 
-        self._import_results(plan)
+        self._compute_target_dvh()
+        self._compute_target_lvh()
 
-    def _import_results(self, plan):
-        self.name = plan.basename
-        self.plan = plan
-        if plan.want_phys_dose:
-            dos_path = os.path.join(plan.working_dir, plan.basename + '.phys.dos')
-            self._import_dos(dos_path)
-            self._compute_target_dvh()
-
-        if plan.want_dlet:
-            let_path = os.path.join(plan.working_dir, plan.basename + '.dosemlet.dos')
-            self._import_let(let_path)
-            self._compute_target_lvh()
-
-    def _import_dos(self, dos_path):
-        logger.debug("Open DosCube {:s}".format(dos_path))
-        dos = pt.DosCube()
-        dos.read(dos_path)
-        self.dose = dos
-
-    def _import_let(self, let_path):
-        logger.debug("Open LETCube {:s}".format(let_path))
-        let = pt.LETCube()
-        let.read(let_path)
-        self.let = let
+    def get_doses(self):
+        return self.plan.dosecubes
+
+    def get_dose(self, dose_type):
+        if dose_type not in ("phys", "bio"):  # TODO: More dose types
```

Comment:
create class variable that holds types

Reply:
Full list of dose types is here: https://github.com/pytrip/pytrip/blob/master/pytrip/dos.py#L54